DOC-3008 Fix consistency with db name and db id variables in examples. (#2989)
RedisLabs/redislabs-docs: content/rs/references/compatibility/resp.md

        rladmin tune db db:<ID> resp3 enabled
        ```

        You may use the database name in place of `db:<ID>` in the above command.

    - [Update database configuration]({{<relref "/rs/references/rest-api/requests/bdbs#put-bdbs">}}) REST API request:

        ```sh

>         PUT /v1/bdbs/<db_name_or_db_id> 

Reviewer: This also only worked with DB ID for me, and attempting to use DB name failed.

    rladmin tune db db:<ID> resp3 disabled
    ```

>     You may use the database name in place of `db:<ID>` in the above command.

Reviewer: ```suggestion
    You can use the database name in place of `db:<ID>` in the preceding command.
```

- [Update database configuration]({{<relref "/rs/references/rest-api/requests/bdbs#put-bdbs">}}) REST API request:

    ```sh

>     PUT /v1/bdbs/<db_name_or_db_id> 

Reviewer: Same problem here, where only DB ID worked for me.
